src/components/SettingMenu/SettingMenu.tsx

```diff
@@ -65,6 +67,14 @@ const SettingMenu = ({ showLogout, logout }: SettingMenuProps) => {
             link="https://dashboard.etherspot.io"
             icon={<MdOutlineDashboardCustomize size={16} style={{ marginRight: '12px' }} />}
           />
+          <MenuItem
+            icon={<BiUser size={16} style={{ marginRight: '12px' }} />}
```

Comment:
we can just put `marginRight: 1`, not a big thing though

Reply:
You mean adding marginRight: '1px' right?
I have added it as per the other items in the settings menu (which has 12px). It leaves way to small space between icon and title with 1px.